### PR TITLE
e2e, passt: Extend test to cover explicit MAC and PCI addresses setting

### PR DESCRIPTION
**Pull request by ormergi**
**What this PR does / why we need it**:
This PR extend passt e2e test to cover MAC and PCI addresses settings 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/network/vmi_passt.go [hunk 1]
```diff
@@ -62,8 +62,16 @@ var _ = SIGDescribe("[Serial] Passt", decorators.PasstGate, Serial, func() {
 	Context("VirtualMachineInstance with passt binding mechanism", func() {
 
 		It("should report the IP to the status", func() {
```
Comment (EdDev) [inline tests/network/vmi_passt.go:64]: This is no longer reflecting what the test does.
Perhaps "should apply the interface configuration" is a better fit now.

**Review by EdDev — COMMENTED** [review]
Nice, thanks.

File: tests/network/vmi_passt.go (hunk 1 above)
Comment (ormergi) [reply tests/network/vmi_passt.go]: Done

**Review by EdDev — APPROVED** [review]

**Pull request merged**